pandas/core/generic.py

```diff
@@ -4350,58 +4350,57 @@ def sample(self, n=None, frac=None, replace=False, weights=None,
 
         Returns
         -------
-        A new object of same type as caller.
+        Series or DataFrame:
+            A new object of same type as caller.
+
+        See Also
+        --------
+        numpy.choice: Generates a random sample from a given 1-D numpy array.
 
         Examples
         --------
         Generate an example ``Series`` and ``DataFrame``:
 
-        >>> s = pd.Series(np.random.randn(50))
-        >>> s.head()
-        0   -0.038497
-        1    1.820773
-        2   -0.972766
-        3   -1.598270
-        4   -1.095526
-        dtype: float64
-        >>> df = pd.DataFrame(np.random.randn(50, 4), columns=list('ABCD'))
+        >>> df = pd.DataFrame({'A': range(0, 500, 10),
+        ...                    'B': range(0, 100, 2),
+        ...                    'C': range(0, 250, 5)})
```

Comment:
I don't think we need so much data, and being arbitrary. Something like the animal datasets we've been using with 6 or 8 rows should be better to illustrate the resample method.